src/components/BarChart/BarChart.jsx

```diff
@@ -237,6 +254,11 @@ const BarChart = createClass({
 			.domain([yAxisMin, yAxisMax])
 			.range([innerHeight, 0]);
 
+		const xFinalFormatter = xAxisFormatter;
+		const yFinalFormatter = yAxisFormatter
```

Comment:
i think `yFinalFormatter = yAxisFormatter || yScale.tickFormat()` is more concise, but why not move both of these into the default values of the desctructure?